src/components/common/FileUpload.js

```diff
@@ -1,40 +1,41 @@
 import React, { useState } from 'react';
-import axios from 'axios';
-import { Button } from 'antd';
+// import axios from 'axios';
```

Comment:
why is this commented out? Does it need to be there? If so please leave a comment next to it as to why its commented out.